src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -64,7 +65,7 @@ The .NET Foundation licenses this file to you under the MIT license.
       <NetCoreAppNativeLibrary Include="System.Native" />
       <NetCoreAppNativeLibrary Include="System.Globalization.Native" Condition="'$(StaticICULinking)' != 'true' and '$(InvariantGlobalization)' != 'true'" />
       <NetCoreAppNativeLibrary Include="System.IO.Compression.Native" />
-      <NetCoreAppNativeLibrary Include="System.Net.Security.Native" />
+      <NetCoreAppNativeLibrary Include="System.Net.Security.Native" Condition="'$(_linuxLibcFlavor)' != 'bionic'" />
```

Comment:
If I am reading the correctly, uses of System.Net.Security.Native are not ifdefed out for linux-bionic in 
https://github.com/dotnet/runtime/blob/680043eca3595352930235fbf9353e1fb9e1f174/src/libraries/System.Net.Security/src/System.Net.Security.csproj#L16

Do we need a matching fix in the libraries too?

Reply:
I don't know. Looking at the latest Bionic Mono build, we're not building this library there either so this can't be a NativeAOT specific problem, if it is a problem:

https://dnceng.visualstudio.com/7ea9116e-9fac-403d-b258-b31fcf1bb293/_apis/build/builds/2202215/logs/1188

```
2023-06-16T01:47:31.0970341Z   Install the project...
2023-06-16T01:47:31.1000197Z   -- Install configuration: "RELEASE"
2023-06-16T01:47:31.1000861Z   -- Installing: /__w/1/s/artifacts/bin/native/net8.0-linux-Release-arm64/./libSystem.IO.Compression.Native.so.dbg
2023-06-16T01:47:31.1019420Z   -- Installing: /__w/1/s/artifacts/bin/native/net8.0-linux-Release-arm64/./libSystem.IO.Compression.Native.so
2023-06-16T01:47:31.1028723Z   -- Installing: /__w/1/s/artifacts/bin/native/net8.0-linux-Release-arm64/./libSystem.IO.Compression.Native.a
2023-06-16T01:47:31.1064467Z   -- Installing: /__w/1/s/artifacts/bin/native/net8.0-linux-Release-arm64/./libSystem.Native.so.dbg
2023-06-16T01:47:31.1070650Z   -- Installing: /__w/1/s/artifacts/bin/native/net8.0-linux-Release-arm64/./libSystem.Native.so
2023-06-16T01:47:31.1074222Z   -- Installing: /__w/1/s/artifacts/bin/native/net8.0-linux-Release-arm64/./libSystem.Native.a
2023-06-16T01:47:31.1085002Z   -- Installing: /__w/1/s/artifacts/bin/native/net8.0-linux-Release-arm64/./libSystem.Globalization.Native.so.dbg
2023-06-16T01:47:31.1089764Z   -- Installing: /__w/1/s/artifacts/bin/native/net8.0-linux-Release-arm64/./libSystem.Globalization.Native.so
2023-06-16T01:47:31.1092925Z   -- Installing: /__w/1/s/artifacts/bin/native/net8.0-linux-Release-arm64/./libSystem.Globalization.Native.a
2023-06-16T01:47:31.1101678Z   -- Installing: /__w/1/s/artifacts/bin/native/net8.0-linux-Release-arm64/./libSystem.Security.Cryptography.Native.OpenSsl.so.dbg
2023-06-16T01:47:31.1107872Z   -- Installing: /__w/1/s/artifacts/bin/native/net8.0-linux-Release-arm64/./libSystem.Security.Cryptography.Native.OpenSsl.so
2023-06-16T01:47:31.1114082Z   -- Installing: /__w/1/s/artifacts/bin/native/net8.0-linux-Release-arm64/./libSystem.Security.Cryptography.Native.OpenSsl.a
2023-06-16T01:47:31.1130823Z   /__w/1/s/src/native/libs
```

I currently can't even test this enough to run a hello world so I'm not able to answer this question. It won't be much worse than Mono-Bionic.

Reply:
We may want to open an issue on it since it looks like a problem.

Reply:
Filed #87665. It looks like System.Net.Security tests are disabled on Bionic due to "Timeout on Helix, cannot repro locally".